src/RequestParser.php

```diff
@@ -133,7 +133,7 @@ public function parseBody($content)
                 return;
             }
 
-            if (strtolower($headers['Content-Type']) == 'application/x-www-form-urlencoded') {
+            elseif (strpos(strtolower($headers['Content-Type']), 'application/x-www-form-urlencoded') === 0) {
```

Comment:
Why did you choose to go for `=== 0`over `!== false`. My preference would be the latter, that leaves room for an client messing up and adding a space in front of it. (Just nitpicking a bit here.)